catalog/app/containers/Bucket/PackageTree/RevisionInfo.tsx

```diff
@@ -27,6 +21,7 @@ const useRevisionInfoStyles = M.makeStyles((t) => ({
     display: 'inline-flex',
   },
   mono: {
+    // @ts-expect-error
     fontFamily: t.typography.monospace.fontFamily,
```

Comment:
When we found a way to add types here, it would be useful if all such code will be the same, so we can use search/replace, i.e.:

```ts
fontFamily: (t.typeography as $TSFixMe).monospace.fontFamy,
```

Reply:
typescript will raise an error and refuse to compile if there's no error where it's expected, so it would be easy to find such cases